### DISCUSSION TITLE
facilitating phonologization and other text-based analyses of transcriptions that have been imported

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#471] [Ideas]
Kai Jia has been working with a set of bash then python then perl messy scripts that allow a person to go from an orthographic representation in CHAT style, like:

> MOT:    don't you like your beeby@c &=yells ?

to a clean version with just the stuff that was said:

> don't you like your beeby

to a phonologized version of that (fake-phonologized in this case):

> dAnt ju laik yAr bibI

to a representation in terms of consonants and vowels:

> CVCC  CV CVVC CVC CVCV

to a "lexicon" with tokens & types of word shapes:

> CV 1
> CVC 1
> CVVC 1 
> CVCC  1
> CVCV 1

This is all done with regular expressions, many of which are constant across languages (CHAT -> clean) and others that are specific (clean -> phonologized).

I wonder to what extent we could incorporate some of these as tools that build on child-project standards, so it's easier to process any .cha files that have already been imported.

@alix-bourree, Kai Jia aims to re-use the ACLEW transcripts to this end soon, so your feedback will be welcome!

**Comment by alecristia** [comment]
@LoannPeurey @alix-bourree @w9k2a4i (that's Kai Jia) - this is the idea I put forward when we met, that would help Kai Jia easily analyze all of her remaining datasets (which we have in childproject format already)

**Reply by LoannPeurey** [reply]
Thank you, Alix and KJ have been discussing it I believe. So this should get on its way soon
